### PR TITLE
Add additional context to Ops Manager stories

### PR DESCRIPTION
**Pull request by ljfranklin**
We got some feedback from recent Onboarding weeks that the Ops Manager stories didn't provide enough context. For example, how does Ops Manager relate to cf-deployment? How does Ops Manager relate to BOSH? This PR attempts to provide a little more context around the Why? of Ops Manager as well as show people how to claim a Toolsmiths environment to start playing around with Ops Manager immediately.

~~Marking this as Work In Progress to get a bit more feedback from folks before merging in.~~ I've fixed up the commits based suggestions below, this should be ready to merge.

### CODE REVIEW

File: opsman.prolific [hunk 1]
```diff
@@ -50,15 +196,15 @@ OpsManager is running and you can log in.
 - #pas-releng Slack
 
 ### Resources
-- https://docs.pivotal.io/pivotalcf/2-2/customizing/
+- https://docs.pivotal.io/pivotalcf/2-4/customizing/gcp.html
```
Comment (mcwumbly) [inline opsman.prolific]: Note: You can remove the version number from the URL to have this always redirect to the latest version, e.g.

```
https://docs.pivotal.io/pivotalcf/customizing/gcp.html
```

File: opsman.prolific [hunk 2]
```diff
@@ -50,15 +196,15 @@ OpsManager is running and you can log in.
...
 
 L: opsman
 
 ---
 
 Deploy BOSH Director using Ops Manager
 
-Configure the BOSH GCP tile using this [guide](https://docs.pivotal.io/pivotalcf/2-2/customizing/gcp-om-config.html). Some tips to keep you from getting stuck:
+Configure the BOSH GCP tile using this [guide](https://docs.pivotal.io/pivotalcf/2-4/om/gcp/config-manual.html). Some tips to keep you from getting stuck:
```
Comment (mcwumbly) [inline opsman.prolific]: ditto: 
```
https://docs.pivotal.io/pivotalcf/om/gcp/config-manual.html
```

File: opsman.prolific [hunk 3]
```diff
@@ -71,7 +217,7 @@ After the deploy succeeds, click around and see if you can information about the
 OpsMan successfully deploys a BOSH Director.
 
 ### Resources
-- https://docs.pivotal.io/pivotalcf/2-2/customizing/ops-man.html
+- https://docs.pivotal.io/pivotalcf/2-4/om/gcp/config-manual.html
```
Comment (mcwumbly) [inline opsman.prolific]: ditto:
```
https://docs.pivotal.io/pivotalcf/om/gcp/config-manual.html
```

File: opsman.prolific [hunk 4]
```diff
@@ -1,10 +1,156 @@
...
+
+```
+#!/bin/bash
+lpass show --notes "TODO: Lyle sent an ask ticket for a LastPass folder and should remember to fill this in later"
+```
+You should have been given access to this LastPass folder at the start of the week, let your facilitators know if you still don't have access.
+
+You should now be able to claim an environment using the steps described [here](https://github.com/pivotal/smith/blob/master/EXAMPLES.md).
+We recommend grabbing the latest PAS version so you can play around with all the latest and greatest features.
+Let your facilitators or ask the Toolsmiths team in slack for help if you run into any issues and consider giving the Toolsmiths a Thank You for providing such a valuable service!
```
Comment (davewalter) [inline opsman.prolific]: Looks like you changed what you were going to say mid-sentence.
```suggestion
Talk to your facilitators or ask the Toolsmiths team in slack for help if you run into any issues and consider giving the Toolsmiths a Thank You for providing such a valuable service!
```

**Review by thausler786 — APPROVED** [review]
Looks good to me, merging so we can use it in the next onboarding week, definitely better than what we previously had.

@bwinter had comments about a confusing section but we are going to fix that in a subsequent PR

**Pull request merged**